src/test/java/org/eclipse/cargotracker/application/BookingServiceIT.java

```diff
@@ -150,13 +153,13 @@ public static WebArchive createDeployment() {
         .addAsLibraries(
             Maven.resolver()
                 .loadPomFromFile("pom.xml")
-                .resolve("org.apache.commons:commons-lang3", "com.h2database:h2")
-                .withTransitivity()
+                .resolve("org.apache.commons:commons-lang3")
```

Comment:
Please test to see if this change is really needed. If it is not needed, remove it.